app/views/feeds/_form_main.html.erb

```diff
@@ -76,3 +76,11 @@
     <%= field_help_text t(".help.display_full_episodes_count") %>
   </div>
 </div>
+
+<div class="col-12 mb-4">
+  <div class="form-floating input-group">
+    <%= form.text_field :footer %>
+    <%= form.label :footer %>
+    <%= field_help_text t(".help.footer") %>
```

Comment:
There are some designs (like the screenshot on #1055) that imply this is a wysiwyg editor on the Podcast -> Engagement Settings.  And (I think) some sort of checkbox on each feed, whether or not to include the "dynamic show notes".

But - it seems like you're thinking this will be customizable per-feed, and just a single line of text.

This is a simpler solution, so I'm fine with starting here.  And we'll get some feedback!

Reply:
yeah this is not the dynamic show notes - that I think will make sense as a different thing, which could include dynamic elements and more of a template even

Reply:
this opt-out sort of footer should be more of a set and forget single value, on other platforms it is not editable by the publishers, and customization and control require help tickets

Reply:
+1